Services/Common/OJS.Services.Common/Implementations/PublisherService.cs

```diff
@@ -13,4 +14,8 @@ public PublisherService(IBus bus)
     public Task Publish<T>(T obj)
         where T : class
         => this.bus.Publish(obj);
+
+    public Task Publish<T>(IEnumerable<T> objs)
+        where T : class
+        => this.bus.Publish(objs);
```

Comment:
Why you ditched the `PublishBatch` method? What this does, isn't it the same as the method above?

Reply:
Reverted changes and fixed the naming convention.